### DISCUSSION TITLE
Welcome to voice-based-email-for-blind Discussions!

### DISCUSSION THREAD

**Discussion by hacky1997** [hacky1997/voice-based-email-for-blind#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ankita18yo** [comment]
Can you provide documentation
